src/main/java/step2/domain/lotto/LottoRank.java

```diff
@@ -0,0 +1,34 @@
+package step2.domain.lotto;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+    _3_RANK(3, 5000),
+    _4_RANK(4, 50000),
```

Comment:
```suggestion
    _4_RANK(4, 50_000),
```
반대로 요기는 언더스코어를 사용하면 가독성이 높아지니 참고해주세요ㅎㅎㅎ